NETProvider/src/FirebirdSql.Data.UnitTests/TestsBase.cs

```diff
@@ -109,9 +116,10 @@ public virtual void TearDown()
 				}
 			}
 			connection.Close();
+			connection.Dispose();
```

Comment:
Without this line, there is dangling database connection and database can't be dropped. 
I think it is a bug.
Reproduces only on GdsDatabase.